src/app/modules/submission/encrypt-submission/encrypt-submission.controller.ts

```diff
@@ -586,6 +595,81 @@ export const handleGetEncryptedResponse: RequestHandler<
   )
 }
 
+// Exported as an array to ensure that the handler always a valid submissionId
+export const handleGetEncryptedResponseUsingQueryParams = [
```

Comment:
can we mark this as `@deprecated` and point it to our new api end point? Now that I think about it, we should have marked all our migrated endpoints as deprecated

Reply:
good point and i'll do so! for the pointing, i'll put the absolute path there just for maximum clarity. will do so for currently open PRs by me

Reply:
@karrui actually i think i'll do this on the routes. makes more sense rather than placing it on the controller; if there are new controllers (like what we have here), i'll add the @deprecate but point it to the new controller